storage/series/series.go

```diff
@@ -520,7 +520,7 @@ func (s *dbSeries) Close() {
 	// a long period of time.
 	s.id = nil
 	s.buffer.Reset(s.opts)
-	s.blocks.Close()
+	s.blocks.Reset()
```

Comment:
should this be `blocks.Close()`?

Reply:
Ah disregard, i see you add the series back to the pool below. This needs to be `Reset()`. Maybe add clarifying comment.

Reply:
yeah I can add a comment